mobile/lib/modules/backup/views/backup_controller_page.dart

```diff
@@ -247,9 +293,12 @@ class BackupControllerPage extends HookConsumerWidget {
                 color: Color.fromARGB(255, 220, 220, 220),
               ),
             ),
-            onPressed: () {
-              AutoRouter.of(context).push(const BackupAlbumSelectionRoute());
-            },
+            onPressed: hasExclusiveAccess
```

Comment:
Why do we have this condition?

Reply:
The user must not change the selected albums while the background service is running as both would fight over the Hive box (and a Hive box can only be opened by one thread at a time)